documentation/guides/config.rst

```diff
@@ -1,12 +1,33 @@
 Configuring :program:`pdns-sysrepo`
 ===================================
-The software reads a YAML configuration file on startup.
-By default, this file is ``/etc/pdns-sysrepo/pdns-sysrepo.yaml``, but this can be changed using the ``-c`` command line argument.
+This software is configured with sysrepo under the ``/pdns-server:pdns-sysrepo/`` tree.
+As this is in the YANG datastore, these settings can be changed at runtime.
 
-These are the default settings and their descriptions:
+This tree has 2 containers, ``pdns-service`` and ``logging``.
 
-.. literalinclude:: ../../pdns-sysrepo.example.yaml
-   :language: yaml
+Configuring logging
```

Comment:
was the include removed on purpose?

Reply:
yes, as there will be no more YAML config after this PR :)

Reply:
duh!